release.toml

```diff
@@ -0,0 +1,11 @@
+[kuadrant]
+channels = ["alpha"]
```

Comment:
`channel` is OLM specific term. What about `[olm]` section? There can be another section for `helm-chart` to set other optional fields like `appVersion` or maybe others ([Chart.yaml](https://helm.sh/docs/topics/charts/#the-chartyaml-file))

Reply:
Ya we can do that. I would be inclined to do `[kuadrant.olm]` as it is more directly related to the kuadrant section.

Reply:
Up to you.

Some suggestion: `kuadrant` seems redundant.. this project is all about kuadrant :)